Gemfile

```diff
@@ -1,3 +1,4 @@
 source 'https://rubygems.org'
 gemspec :name => 'gollum-lib'
 gem 'irb'
+gem 'gollum-rugged_adapter', :git => 'https://github.com/gollum/rugged_adapter/', :branch => 'find_branches'
```

Comment:
I wonder if it's worth leaving a code comment with a link to the issue inline? That way if more changes are made to this line or this file before `find_branches` is merged, the context needed to understand the issue won't be be three clicks away in a PR description.

Reply:
Will do this for now, but as I said I'd prefer to remove this temp change altogether before merging this into 6.x!